### ISSUE TITLE
Prayer against devils, called masṭǝma gānen 'The Drowner of Devils' in BLorient5423

### ISSUE THREAD

**Issue by CarstenHoffmannMarburg** [BetaMasaheft/Documentation#2603]
@karljonaskarlsson @DenisNosnitsin1970 

I found a prayer in BLorient5423 (magical scroll) that was entitled "Prayer against devils, called masṭǝma gānen 'The Drowner of Devils'" (I would rather translate: 'The drowning of devils'). 

I have not found a LIT ID with this special text, but I found LIT2243salotb, which has no text, no witness and no description beyond the title. I wonder, whether this ID could serve as a generic ID to describe different prayers, in which demon of various kinds are to be drowned. 

I found a number of manuscripts (EMIPms00148, EMIPms00530 and many more), mostly scroll with texts of drowning demons. However, as far as I can see, these texts do not have IDs and they are always different from one another and from BLorient5423. Either the names of the demons are different or the days for which it is arranged are different. In addition BLorient5423 is special insofar, as the word for drowning is placed behind the number of demons which are listed individually in contrast to many others, where the drowning is mentioned in the beginning (as for example in EMIPms00530: ጸሎት፡ በእንተ፡ መሥጥመ፡ አጋንንት፡). 

I think, these prayers can be combined in a generic ID instead of creating a new ID for each single prayer, that contains drowning demons. What do you think? Is it better to create a generic ID or a specific ID for this text in BLorient5423? Do you want me to use LIT2243salotb or should I better create a new LIT record?

```
<msItem xml:id="ms_i1.2">
                                <title ref="LIT00000000000000" type="incomplete">Prayer against devils, called masṭǝma gānen 'The Drowner of Devils'</title>
                                <note>Text written in black and arranged for the days of the week starting with Sunday and ending in the middle of the 
                                    prayer for Tuesday.</note>
                                <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> ጸሎት፡ በእንተ፡ ባርያ፡ ወሌጌዎን፡ ዛር፡ ወነገርጋር፡ ወጋኔነ፡ ቀትር፡ ወእምኵሉ፡ እኩይ፡ 
                                    ግብር፡ ንጽሕፍ፡ እንከ፡ ነገረ፡ መሥጥም፡ ጸሎት፡ ዘዕለተ፡ እሁድ፡ ዓባይ፡ ዘባርያ፡ ወሌጌዎን፡ በአህያ፡ በሻራህያ፡</incipit>
                                <textLang mainLang="gez"/>
                            </msItem>
```

**Comment by DenisNosnitsin1970** [comment]
If there is an absolutely free record, I assume you can use it. As to generic id, I am not sure. If you mean under "generic" all prayers which deal with drowing of demons, irrespective their textual form, this will be a sway away from what we do. We have "general record" for potentially interrelated texts of unclear recensions or shapes, but not for anything else (even though, you are right, If I understand you well: there are many prayers where the terms which we perceive as title and indication of pertinence to a textual tradition, i.e. mesfefaqer, mestenager, mastema ganen, meftahe seray, denotes rather a function and not a title, and may be completely different and unrelated to each other (of course the contrary is not excluded as well)).

**Comment by CarstenHoffmannMarburg** [comment]
> As to generic id, I am not sure. If you mean under "generic" all prayers which deal with drowing of demons, irrespective their textual form, this will be a sway away from what we do. We have "general record" for potentially interrelated texts of unclear recensions or shapes, but not for anything else

I was not aware of the difference between generic and general records, but now I understand better. From my point of view, it would be nice to have (generic) records for all the cases you mention above, for those witnesses where nothing more is known, and individual records for those where the exact text is known. However, my impression is that most magical prayers are written or rewritten ad hoc, with some variation in length, phrasing and number of demons or calamities mentioned, but not copies of another manuscript witness. I do not think it makes sense to create individual IDs for almost every textual witness that is attested only once in that particular form. We need to keep in mind that independent transmission is an important part of our clavis, i.e. the decision for a new ID should be made when the text is attested more than once. I think we do not follow this rule for many magical prayers. In fact, we find very many records without any ID at all, and it would be helpful if there were generic IDs for them, indicating the function rather than the exact text. This would be much easier to encode and more helpful to the user.

**Comment by CarstenHoffmannMarburg** [comment]
Following the discussion on https://github.com/BetaMasaheft/Documentation/issues/2598, I have learned that it is better to encode new texts with individual LIT IDs and mark the similarities to other existing IDs, because it is easier to merge similar or identical texts than to disambiguate them later.

In this case, I will use LIT2243salotb to encode the above text in BLorient5423.

What does @eu-genia say about this question in general and in this case in particular?

**Comment by CarstenHoffmannMarburg** [comment]
@nafisa-valieva

**Comment by nafisa-valieva** [comment]
Thank you very much for making me part of this discussion!I do not have any particular opinion for the moment, but I learnt a lot.

**Comment by eu-genia** [comment]
Again in this case I think what you mean under "generic" record would be an Authority file in taxonomy to be able to classify the spell as the one dealing with the drowning of demons; a taxonomy for magic is a desideratum, I am sure @abausi would agree , but as I wrote here https://github.com/BetaMasaheft/Documentation/issues/2596#issuecomment-2324657113 I wonder if it should be a task for someone interested in the field. I would be against "sporadically" creating keywords, we need a more systematic approach.

As for Clavis IDs - just as in the case of qene - the approach should be, when we have text to be able to identify the text, we need to create an individual CAe ID for each prayer, if not already there. 

We can use a "general" CAe ID LIT6609ProtectPrayer for a not further identifiable magic text if no text is provided/available.

**Comment by eu-genia** [comment]
Thank you!
